app.py

```diff
@@ -204,7 +205,76 @@ def calculate_minutes(departure):
     else:
         return f'Ca {countdown}'
 
-
+def get_temperature():
+    if temp_situation["updated"] == 0 or temp_situation["updated"] < datetime.now(tz.gettz("Europe/Stockholm")) - timedelta(minutes=10):
+        print("Updating temperatures")
+        openweathermap_key = os.environ["OWM_KEY"].strip()
```

Comment:
I thing you need a way to load the environment variable from a file. 
Something like this by the init stuff
`if "OWM_KEY" not in list(os.environ.keys()):
    with open("creds.txt") as f:
        owm_key = f.readlines()[-1]
    os.environ["OWM_KEY"] = owm_key`